exchange/bidder.go

```diff
@@ -328,11 +328,6 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, bidderRequest Bidde
 				if err == nil {
 					// Conversion rate found, using it for conversion
 					for i := 0; i < len(bidResponse.Bids); i++ {
-						if bidResponse.Bids[i].BidMeta == nil {
-							bidResponse.Bids[i].BidMeta = &openrtb_ext.ExtBidPrebidMeta{}
-						}
-						bidResponse.Bids[i].BidMeta.AdapterCode = bidderRequest.BidderName.String()
```

Comment:
This is the code which conflicts with `if prebid.Meta == nil && maputil.HasElement(extMap, "prebid", "meta") {` check in exchange.go? I can see that as being a legit bug. Nice find.

Reply:
Yep exactly, since prebid.Meta is set here it never enters that other block which checks for `if prebid.Meta == nil` which is responsible for building the rest of prebid.Meta

Reply:
Sounds good. You have the go-ahead.

Reply:
Great - I'll continue with this PR